File: Python/Problem_7.py
def Problem_7():
    i = 2
    curNum = 3
    l = []
    while(i != 10001):
        curNum+=2
        i+=1
        for j in range(2,int(curNum**0.5)+1):
            if(curNum%j==0):
                i-=1
                break

    return curNum

print(Problem_7())
#ANSWER: 104743

File: C++/Problem_7.cpp
#include <iostream>
using namespace std;


int Problem_7() {
	int i = 2;
	int num = 3;
	while (i != 10001) {
		num+=2;
		for (int j = 2; j <= int(sqrt(num)) + 1; j++) {
			if (num % j == 0) {
				break;
			}
			else if (j == int(sqrt(num)) + 1) {
				i++;
			}
		}
	}
	return num;
}

int main() {
	
	cout << Problem_7() << endl;
	return 0;
}
//ANSWER: 104743
